src/main/java/org/chzzk/howmeet/domain/regular/schedule/controller/MSController.java

```diff
@@ -0,0 +1,33 @@
+package org.chzzk.howmeet.domain.regular.schedule.controller;
+
+import lombok.RequiredArgsConstructor;
+import org.chzzk.howmeet.domain.regular.schedule.dto.MSRequest;
+import org.chzzk.howmeet.domain.regular.schedule.dto.MSResponse;
+import org.chzzk.howmeet.domain.regular.schedule.service.MSService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RequiredArgsConstructor
+@RequestMapping("/member-schedule")
+@RestController
+public class MSController {
+    private final MSService msService;
+
+    @PostMapping
+    public ResponseEntity<?> createMemberSchedule(@RequestBody final MSRequest msRequest) {
+        final MSResponse msResponse = msService.createMemberSchedule(msRequest);
+        return ResponseEntity.ok(msResponse);
+    }
```

Comment:
생성 API의 경우 아래와 같이 상태 코드를 201, 헤더에 조회 URL 을 제공해주는건 어떨까요?

```suggestion
    @PostMapping
    public ResponseEntity<?> createMemberSchedule(@RequestBody final MSRequest msRequest) {
        final MSResponse msResponse = msService.createMemberSchedule(msRequest);
        returnreturn ResponseEntity.created(URI.create("/member-schedule/" + msResponse.id()))
                .body(msResponse);
        )
    }
```